src/services/questionnaireAnalyzer/questionnaireAnalyzer.js

```diff
@@ -398,63 +398,44 @@ const checkCompletionStateOfMultipleItems = (items, props) => {
  * @param  {QuestionnaireItem} [item] questionnaire item
  */
 const checkDependenciesOfSingleItem = item => {
-
 	/** return value of the function, true if all dependencies are met */
-	let returnValue = false
+	let returnValue = false //TODO: Why not directly use "return false;" or "return true;" ????
```

Comment:
Yeah i think, instead of assigning to this variable we could directly return in the respective if/else branch @GolexPrime

Reply:
Yeah, it's a remnant from a different time in development where directly returning the value wasn't an option.
BTW - nice touch with the 4 exclamation marks...